src/main/java/edu/gatech/cs2340/trydent/animation/DispatchAnimationListener.java

```diff
@@ -0,0 +1,83 @@
+package edu.gatech.cs2340.trydent.animation;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class DispatchAnimationListener implements AnimationListener {
```

Comment:
Since this is public, add some description that this class is a container for animation listeners?